satellite/metabase/rangedloop/observerstats.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2022 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package rangedloop
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/spacemonkeygo/monkit/v3"
+)
+
+func sendObserverDurations(observerDurations []ObserverDuration) {
+	initCompletedObserverStats()
+	completedObserverStatsInstance.setObserverDurations(observerDurations)
+}
+
+// completedObserverStatsInstance is initialized once
+// so that hopefully there is never more than once objec instance per satellite process
```

Comment:
nit: objec -> object

Reply:
I'll change this in another PR so I don't lose the approve.